### DISCUSSION TITLE
Out of the box OTA code not working

### DISCUSSION THREAD

**Discussion by johnsmithecon** [Mair/esp32-course#177] [Course Qeustions] [answered]
Hi all! I'm trying to get a simple OTA update working with the ESP32. I followed the tutorials on the OTA updates from the course, but when I try with google drive I get a variety of issues. An example can be seen in the image below. How can I run a simple and functional OTA program, and has anyone been able to successfully do this recently? Thanks!

<img width="1201" alt="image" src="https://user-images.githubusercontent.com/62140912/187367679-25d22e26-af5c-48a0-bbe1-17bb781dd07a.png">

**Comment by Mair** [comment, marked as answer]
Looks to me like a certificate issue.
What is the URL you to your bin? (Getting the URL can be a bit tricky)
Perhaps try to clone the repo and see if yo can run my original example?

**Reply by Mair** [reply]
If you are still struggling lets see your code

**Reply by johnsmithecon** [reply]
I will try with AWS S3. I think it's a certification issue with google drive. Thanks Mair!
